modules/auxiliary/scanner/http/wordpress_scanner.rb

```diff
@@ -126,7 +128,37 @@ def run_host(target_host)
         end
         print_status("#{target_host} - Finished scanning plugins")
       end
-      print_status("#{target_host} - Finished all scans")
+      #      if 1==1
+      if datastore['USERS']
+        print_status("#{target_host} - Searching Users")
+        url = wordpress_url_rest_api + '/users'
+        res = send_request_cgi({
+          'method' => 'GET',
+          'uri' => url
+        })
```

Comment:
We typically encourage using `normalize_uri` to concatenate strings for this purpose.

```suggestion
        res = send_request_cgi({
          'method' => 'GET',
          'uri' => normalize_uri(wordpress_url_rest_api,  'users')
        })
```

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16061/commits/2d9edcd22f45cfe495188cbb0007eeee4b69cf25